config/locales/views/saving_goals/es.yml

```
@@ -0,0 +1,93 @@
+      target: "Objetivo: %{amount}"
+      deadline: "Fecha límite: %{date}"
+      on_track: "A tiempo"
+      behind: "Con retraso"
+      contributions: "Contribuciones"
+      actions: "Acciones"
+      edit: "Editar objetivo"
+      delete: "Eliminar objetivo"
+      target_date: "Fecha objetivo: %{date}"
+      current_balance: "Saldo actual"
+      of_target: "del objetivo de %{amount}"
+      on_track: "A tiempo"
```

YAMLlint (1.38.0)

[error] duplication of key "on_track" in mapping

(key-duplicates)